src/cmds/mod.rs

```diff
@@ -0,0 +1,525 @@
+use crate::{
+    hex_field::{self, HexNetID},
+    region::Region,
+    DevaddrConstraint, Msg, PrettyJson, Result,
+};
+use anyhow::Context;
+use clap::{Args, Parser, Subcommand};
+use helium_crypto::PublicKey;
+use std::path::PathBuf;
+
+pub mod env;
+pub mod org;
+pub mod route;
+
+pub const ENV_CONFIG_HOST: &str = "HELIUM_CONFIG_HOST";
+pub const ENV_KEYPAIR_BIN: &str = "HELIUM_KEYPAIR_BIN";
+pub const ENV_NET_ID: &str = "HELIUM_NET_ID";
+pub const ENV_OUI: &str = "HELIUM_OUI";
+pub const ENV_MAX_COPIES: &str = "HELIUM_MAX_COPIES";
+
+#[derive(Debug, Parser)]
+#[command(name = "helium-config-cli")]
+#[command(author, version, about, long_about=None)]
+pub struct Cli {
+    #[command(subcommand)]
+    pub command: Commands,
+
+    #[arg(
+        global = true,
+        long,
+        env = ENV_CONFIG_HOST,
+        default_value = "http://50.18.149.124:50051"
```

Comment:
this a dummy value or does it make sense to make this default value a localhost address or something unless there's a permanent dns record we could set it to?

Reply:
This was the alpha testing server. 
It was hardcoded to make it a bit easier for 3rd party testing. 
It will be changed in the future.